src/text/text.hpp

```diff
@@ -220,8 +227,10 @@ namespace text {
 		alignment align = alignment::left;
 		text_color color = text_color::white;
 	};
+
 	struct layout {
 		std::vector<text_chunk> contents;
+		std::vector<layout_divider> dividers;
```

Comment:
This won't work if the layout needs to rearrange the text, for example if the layout turns into more than one column